change/@ni-nimble-components-14b23480-337b-4f84-a769-036ac4a6565c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add a \"large mode\" option for the nimble-dialog which simply changes its width to 1024px and its height to 680px",
+  "packageName": "@ni/nimble-components",
+  "email": "diana.anca@ni.com",
```

Comment:
If you'd like to hide your NI email address from the world, you can [configure Github to keep your email address private](https://github.com/settings/emails). After you make this change you'd have to manually modify this file to use your private address but for future PRs it should use the private address automatically.